applications/app_test/app_uartpb.cpp

```diff
@@ -0,0 +1,34 @@
+#include "app.hpp"
```

Comment:
in commit `platforms: move Protobuf messages handler from platform to application`:
all types of Protobuf messages can be received. => all types of Protobuf messages **that** can be received.